ts/tsconfig.json

```diff
@@ -0,0 +1,104 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
This is not JSON, as the format does not allow for comments. Can we skip such file and use explicit command arguments (in Makefile) instead?